### ISSUE TITLE
https://github.com/MLWave/Black-Boxxy/blob/master/eli.py used for lightGBM classificatoin?

### ISSUE THREAD

**Issue by Sandy4321** [MLWave/Black-Boxxy#1]
great post thanks
but
can be 
https://github.com/MLWave/Black-Boxxy/blob/master/eli.py
used for lightGBM classificatoin?

**Comment by MLWave** [comment]
Yes. ELI5 supports LightGBM. May need to change some references. This should help: https://eli5.readthedocs.io/en/latest/autodocs/lightgbm.html

**Comment by MLWave** [comment]
As an addition, if you want interpretability for LightGBM, I suggest using the library https://github.com/interpretml/interpret (which is far more solid than the experiments done here).
